model_definition.md

```diff
@@ -64,7 +64,8 @@ The total population consists of $K_\mathrm{total}$ subpopulations $k$ with corr
 
 Whenever the sum of the wastewater catchment population sizes $\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k$ is less than the total population size $n$, we use an additional subpopulation of size $n - \sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k$ to model individuals in the population who are not covered by wastewater sampling.
 
-The total number of subpopulations is then $K_\mathrm{total} = K_\mathrm{sites} + 1$: the $K_\mathrm{sites}$ subpopulations with sampled wastewater, and the final subpopulation to account for individuals not covered by wastewater sampling.
+The total number of subpopulations is then $K_\mathrm{total} = K_\mathrm{sites} + 1$: the subpopulation to account for individuals not covered by wastewater sampling plus the $K_\mathrm{sites}$ subpopulations with sampled wastewater. We refer to first subpopulation as the reference subpopulation, denoted by $k=0$.
+
 The model without wastewater (hospital admissions only model) is therefore a special case of the model where $K_\mathrm{sites} = 0$ and $K_\mathrm{total} = 1$, with subpopulation size $n_k = n$, the total population.
 In the case where the sum of the wastewater site catchment populations meets or exceeds the total population ($\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k \ge n$) the model does not use a final subpopulation without sampled wastewater. In that case, the total number of subpopulations $K_\mathrm{total} = K_\mathrm{sites}$.
```

Comment:
Need to be consistent between "first" and "final" subpopulation, and state explicitly which subpopulation is the reference subpopulation when $\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k \ge n$